lib/Runtime/Base/JnDirectFields.h

```diff
@@ -80,6 +80,7 @@ ENTRY(clz32)
 ENTRY(fround)
 // End math functions
 
+ENTRY(Promise)
```

Comment:
I was just removing Entry(getVarDate) but did a quick review and noticed many other Entrys here that were no longer required hence deleted a lot and re-ordered some to make it neater.

Note: this file is used to populate internal PropertyIds and BuiltInPropertyRecords.